main/zoe/guide/contracts/atomic-swap.md

```diff
@@ -2,6 +2,9 @@
 
 <Zoe-Version/>
 
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/958a2c0a3dec38bdba2234934119ea2c28958262/packages/zoe/src/contracts/atomicSwap.js) (Last updated: 4/22/2020)
```

Comment:
Hmm, I'm not sure why this is showing up, since we already have these links on master.